h/tweens.py

```diff
@@ -96,3 +97,20 @@ def content_security_policy_tween(request):
         return resp
 
     return content_security_policy_tween
+
+
+def redirect_tween_factory(handler, registry):
+    redirects = {
```

Comment:
I'd suggest you move this into a module constant, which is then a default argument to the factory function:

```python
REDIRECTS = {...}

def redirect_tween_factory(handler, registry, redirects=REDIRECTS):
    ...
```

Then you can add some tests for this function :)